index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls, or a <a>holder</a>
```

Comment:
I am more comfortable with this language; however, leaving how an issuer determines that a subject controls such a method seems like a possible footgun. Won't block my approval - just a call out.

Reply:
We updated the language to require the issuer to check that a holder can satisfy each `confirmationMethod`. It essentially boils down to the issuer making a trustworthy claim.